src/mods/common/analyzer/append/bpy.types.mod.rst

```diff
@@ -50,13 +50,6 @@
       :mod-option rtype: skip-refine
       :option function: overload
```

Comment:
Should we remove `overload` option for each method?

Reply:
No, we're still defining more than one version of each method, so we need the `@overload` decorator.

Reply:
Thank you for your explanation.
I understood.